modules/shadertools/src/index.ts

```diff
@@ -93,7 +93,7 @@ export {pbrMaterial} from './modules/lighting/pbr-material/pbr-material';
 // DEPRECATED - v8 legacy shader modules (non-uniform buffer)
 
 // math libraries
-export {fp64, fp64arithmetic} from './modules-webgl1/math/fp64/fp64';
+// export {fp64, fp64arithmetic} from './modules-webgl1/math/fp64/fp64';
```

Comment:
Can we cut instead? At least on master which is targeting 9.2

Reply:
I'm in favour of removing all the old `modules-webgl1` code & exports. I think it has all been ported